motr/examples/example2.c

```diff
@@ -0,0 +1,253 @@
+/* -*- C -*- */
+/*
+ * Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * For any questions about this software or licensing,
+ * please email opensource@seagate.com or cortx-questions@seagate.com.
+ *
+ */
+
+/*
+ * Example Motr application to create index, put key&val into index,
+ * get val from index, and then delete the index.
+ */
+
+/*
+ * Please change the dir according to you development environment.
+ *
+ * How to build:
+ * gcc -I/work/cortx-motr -I/work/cortx-motr/extra-libs/galois/include \
+ *     -DM0_EXTERN=extern -DM0_INTERNAL= -Wno-attributes               \
+ *     -L/work/cortx-motr/motr/.libs -lmotr                            \
+ *     example2.c -o example2
+ *
+ * Please change the configuration according to you development environment.
+ *
+ * How to run:
+ * LD_LIBRARY_PATH=/work/cortx-motr/motr/.libs/                              \
+ * ./example2 172.16.154.179@tcp:12345:34:1 172.16.154.179@tcp:12345:33:1000 \
+ *         "<0x7000000000000001:0>" "<0x7200000000000001:64>" 12345670
+ */
+
+#include "motr/client.h"
+#include "lib/trace.h"
+#include <stdio.h>
+#include <stdlib.h>
+#include <errno.h>
+
+enum {
+	KV_COUNT = 10,
+	KV_LEN   = 32,
+};
+
+static int index_launch_wait(struct m0_entity *e,
```

Comment:
There is nothing index-specific in this function. I'd rather name it like "op_wait_fini()" or something.

Reply:
OK